dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/cloudera/manager/dto/ApiClusterDto.java

```diff
@@ -0,0 +1,134 @@
+/*
+ * Copyright 2022-2024 Google LLC
+ * Copyright 2013-2021 CompilerWorks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.edwmigration.dumper.application.dumper.connector.cloudera.manager.dto;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.util.List;
+
+/**
+ * DTO class for the official Cloudera API <a
+ * href="https://archive.cloudera.com/cm7/7.11.3.0/generic/jar/cm_api/apidocs/json_ApiCluster.html">json</a>
+ *
+ * <p>Note: The license for <a
+ * href="https://mvnrepository.com/artifact/com.cloudera.api.swagger/cloudera-manager-api-swagger/7.11.0">generated</a>
+ * code is unclear, the own model for public schema used instead of it.
```

Comment:
I don't understand this comment.

Also, is this class generated?

Since we use Jackson's ObjectMapper to deserialize this, it might be worth using AutoValue and Builder, which I think Jackson can honour. It will greatly reduce the amount of "boring" code here: https://stackoverflow.com/questions/43364266/jackson-autovalue-builder-bean-validation

Reply:
The classes are created according to the schema, not generated by a tool. I will play with autovalue+Jackson in background, if it will reduce boilerplate I will adopt it here.